src/composables/useNotifications.ts

```diff
@@ -1,20 +1,30 @@
 import { ref } from 'vue';
 
 interface Notification {
+  id: number;
   message: string;
   type: string;
-  timestamp: number;
+  remove(): any;
```

Comment:
is this a new way to declare a function?

Reply:
Looks fine otherwise

Reply:
Like mentioned before, I'm not too familiar with Typescript. This is what I found in examples, like here: https://www.typescriptlang.org/docs/handbook/interfaces.html

```ts
interface ClockInterface {
  tick(): void;
}
```

Reply:
Good to know that we can define functions like this 

`tick(): void`

I always used `tick = () => void`